index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <h1> INTENTION TIMER </h1>
+  </header>
+  <main>
+    <section class="left-section">
+      <h2 class="new-activity" New Activity </h2>
+        <section class="left-section">
+          <p class="select-category">Select a category:</p>
+          <button class="study-button" id="studyButton" type="button">Study</button>
+          <button class="meditate-button" id="meditateButton" type="button">Meditate</button>
+          <button class="exercise-button" id="exerciseButton" type="button">Exercise</button>
```

Comment:
Do we need to add a group to all the buttons like with radio buttons so you can't click more than one?